react/features/feedback/components/FeedbackButton.web.js

```diff
@@ -0,0 +1,28 @@
+/* global config */
+import React, { Component } from 'react';
+
+/**
+ * A Web Component which renders feedback button.
+ */
+export class FeedbackButton extends Component {
```

Comment:
GH swallowed my comment so I'm going for it again :-)

> Since this component doesn't use any of the lifecycle methods nor state you can make it a stateless functional component instead.